utils/frame/benchmarking-cli/src/storage/weights.hbs

```diff
@@ -66,7 +65,7 @@ pub mod constants {
 	#[cfg(test)]
 	mod test_db_weights {
 		use super::constants::{{db_name}}Weight as W;
-		use frame_support::weights::constants;
+		use sp_weights::constants;
```

Comment:
should we just re-export the same stuff through frame support for backwards compat?